### DISCUSSION TITLE
Scrolling window management?

### DISCUSSION THREAD

**Discussion by carnivorecookies** [qtile/qtile#5070] [Ideas]
Scrolling window management is a really nice workflow that removes some of the downsides of tiling. Windows can extend past the monitor itself and you can scroll to the end of it.

An example of this would be Niri- https://github.com/YaLTeR/niri)](https://github.com/YaLTeR/niri
Or PaperWM- https://github.com/paperwm/PaperWM

**Comment by elParaguayo** [comment]
I started working on a scrolling layout. Ran into a few issues and got distracted. I could probably pick it up again...
